ssn/chapters/Sampling.html

```diff
@@ -60,9 +60,7 @@ <h5><a href="#SOSASample">sosa:Sample</a></h5>
         </p>
         <p>
             <em>Sample</em> &mdash;
-            Feature which is intended to be representative of a
-            <a href="#SOSAFeatureOfInterest">FeatureOfInterest</a> on which
-            <a href="#SOSAObservation">Observations</a> may be made.
+            feature which is intended to be representative of a FeatureOfInterest on which Observations may be made.
```

Comment:
Should these not link via ReSpec?
```suggestion
            feature which is intended to be representative of a <a>FeatureOfInterest</a> on which <a>Observations</a> may be made.
```

Reply:
In #340 I put <dfn> tag around the CURI, so ReSpec will only link if namespace prefix (`sosa:`) is present. 

Furthermore, I removed links from the text definitions. 
There is no shortage of links everywhere else!